library/src/main/java/com/okta/appauth/android/OktaAppAuth.java

```diff
@@ -199,24 +256,29 @@ public void signOutFromOkta(
             final PendingIntent completionIntent,
             final PendingIntent cancelIntent
     ) {
-        if (isUserLoggedIn()) {
-            mExecutor.submit(new Runnable() {
-                @Override
-                public void run() {
-                    doEndSession(
-                            OktaManagementActivity.createStartIntent(
-                                    context.getApplicationContext(),
-                                    completionIntent,
-                                    cancelIntent),
-                            cancelIntent);
-                }
-            });
-        } else {
+        if (!isUserLoggedIn()) {
             throw new IllegalStateException("No logged in user found");
         }
+        mExecutor.submit(new Runnable() {
+            @Override
+            public void run() {
+                doEndSession(
+                        OktaManagementActivity.createStartIntent(
+                                context.getApplicationContext(),
+                                completionIntent,
+                                cancelIntent),
+                        cancelIntent);
+            }
+        });
+
     }
 
-    void clearSessionData() {
+    /**
+     * Removes all stored information opn current session like
+     * Tokens and Authentication Server config.
+     * NOTE: After removal {@link OktaAppAuth#init} should be called.
+     */
+    public void clearSessionData() {
```

Comment:
Let's name this `clear()` to match https://github.com/okta/okta-sdk-appauth-ios#clear

Reply:
I don’t think we should try to match iOS method names here.  ’oktaAppAuth.clear()‘ could have multiple meanings. I think ‘clearSession’ or something like ‘endSession’ would be a better fit.   That said if the actual class name was something like ‘OktaSession’, cleat would be perfect.
We probably need to think about removing “AppAuth” from these class names before 1.0 anyway.
Thoughts?

Reply:
I like **clearSession** more. Also 'AppAuth' looks good as a name of the SDK entry point, but if we want to move to this name I would also consider removing AppAuth-Android as a dependency.

Reply:
"AppAuh" also implies the use the AppAuth project, if that project becomes an implementation detail IMHO we may want to change the name of this class 🤷‍♂️, but we can worry this class name later

Reply:
I meant OktaAuth, sorry )